src/sagemaker/tensorflow/README.rst

```diff
@@ -614,188 +612,25 @@ Note that TensorBoard is not supported when passing wait=False to ``fit``.
-      ]}
-    ]}
-  }
-
-Specifying the output of a prediction request
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The format of the prediction ``result`` is determined by the parameter ``export_outputs`` of the `tf.estimator.EstimatorSpec <https://www.tensorflow.org/api_docs/python/tf/estimator/EstimatorSpec>`_ that you returned when you created your ``model_fn``, see
-`Example of a complete model_fn`_ for an example of ``export_outputs``.
-
-More information on how to create ``export_outputs`` can find in `specifying the outputs of a custom model <https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/docs_src/programmers_guide/saved_model.md#specifying-the-outputs-of-a-custom-model>`_.
-
-Endpoint prediction request handling
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Whenever a prediction request is made to a SageMaker Endpoint via a ``InvokeEndpoint`` SageMaker operation, the request will
-be deserialized by the web server, sent to TensorFlow Serving, and serialized back to the client as response.
-
-The TensorFlow Web server breaks request handling into three steps:
-
--  input processing,
--  TensorFlow Serving prediction, and
--  output processing.
-
-The SageMaker Endpoint provides default input and output processing, which support by default JSON, CSV, and protobuf requests.
-This process looks like this:
-
-.. code:: python
-
-    # Deserialize the Invoke request body into an object we can perform prediction on
-    deserialized_input = input_fn(serialized_input, request_content_type)
-
-    # Perform prediction on the deserialized object, with the loaded model
-    prediction_result = make_tensorflow_serving_prediction(deserialized_input)
-
-    # Serialize the prediction result into the desired response content type
-    serialized_output = output_fn(prediction_result, accepts)
-
-The common functionality can be extended by the addiction of the following two functions to your training script:
-
-Overriding input preprocessing with an ``input_fn``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-An example of ``input_fn`` for the content-type "application/python-pickle" can be seen below:
-
-.. code:: python
-
-    import numpy as np
-
-    def input_fn(serialized_input, content_type):
-        """An input_fn that loads a pickled object"""
-        if request_content_type == "application/python-pickle":
-            deserialized_input = pickle.loads(serialized_input)
-            return deserialized_input
-        else:
-            # Handle other content-types here or raise an Exception
-            # if the content type is not supported.
-            pass
-
-Overriding output postprocessing with an ``output_fn``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-An example of ``output_fn`` for the accept type "application/python-pickle" can be seen below:
-
-.. code:: python
 
-    import numpy as np
+- The second option uses a TensorFlow Serving-based server to provide a super-set of the
```

Comment:
should this be "TensorFlow-Serving-based"?

Reply:
don't think so. looks odd.